ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePage.tsx

```diff
@@ -169,9 +169,6 @@ function ImagePage() {
                                             <GenerateSbomModal
                                                 onClose={() => setSbomTargetImage(undefined)}
                                                 imageName={sbomTargetImage}
-                                                onGenerateSbom={() => {
-                                                    // TODO Implement analytics event
-                                                }}
```

Comment:
I opted to handle this in the modal instead of passing the same callback to both locations. IMO the action being performed by the modal, which encapsulates the same API call in both cases, is specific enough that we don't need to customize behavior at this point.